src/MainPage/Components/Navigation/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
Prebaciti sa dvostrukih navodnika importe na jednostruke kao sto je to do sata bila praksa u projektu
```suggestion
import React from 'react';
```